app/models/game.rb

```diff
@@ -36,6 +36,17 @@ def check?
     kings_in_check?('white') || kings_in_check?('black')
   end
 
+  def forfeit(current_user)
+    if current_user.id == white_player_id
+      white_player_id.update_attributes(white_player_id.losses: white_player_id.losses + 1)
```

Comment:
Also, you could use `increment`. Which looks even nicer.

http://apidock.com/rails/ActiveRecord/Persistence/increment